docker/include/bin/start_faux

```diff
@@ -277,5 +277,8 @@ else
     echo Runtime config $conf_file not found.
 fi
 
+# Trigger wpasupplicant
```

Comment:
this should be configurable -- see all the places where it says ${options -- I'd just use wpa -- so ${options[wpa]} should suffice.  (Also -- comment isn't helpful -- it's inherently obvious what the code does, so it's just extra words!)

Reply:
Changed.